File: Python Code/Daily_Flash_Code_0/Week10/25Mar/56-DailyFlash_Solutions/25_Mar_Solution_One/CPP/Program0.cpp
#include <stdio.h>
#include <math.h>

int main() {

	int l = 0,x = 0,y = 0;
	float sum = 0;
	printf("Enter The Length of the Series\n");
	scanf("%d",&l);
	
	printf("Enter value of x\n");
	scanf("%d",&x);
	
	printf("Enter value of y\n");
	scanf("%d",&y);

	x = x + y;
	for(int i = 1; i <= l; i++)
		sum+=(i/(pow(x,i)));
	
	printf("Sum of Series = %.4f\n",sum);

	return 0;
}

File: Python Code/Daily_Flash_Code_0/Week10/25Mar/56-DailyFlash_Solutions/25_Mar_Solution_One/Java/Program0.java
import java.io.*;

class Program0 {


	public static void main(String[] args) throws IOException {

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		float sum = 0;
		int l = 0;
		System.out.printf("Enter The Length of the Series\n");
		l = Integer.parseInt(br.readLine());

		System.out.printf("Enter value of x\n");
		int x = Integer.parseInt(br.readLine());

		System.out.printf("Enter value of y\n");
		int y = Integer.parseInt(br.readLine());
		x = x+y;

		for(int i = 1; i <= l; i++)
			sum+=(i/(Math.pow(x,i)));
		System.out.println("Sum of Series = "+String.format("%.4f",(float)sum));
	}
}
